hack/deploy-karmada-agent.sh

```diff
@@ -7,96 +7,102 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
 function usage() {
   echo "This script will deploy karmada agent to a cluster."
-  echo "Usage: hack/deploy-karmada-agent.sh <KARMADA_APISERVER_KUBECONFIG> <KARMADA_APISERVER_CONTEXT_NAME> <MEMBER_CLUSTER_KUBECONFIG> <MEMBER_CLUSTER_CONTEXT_NAME>"
-  echo "Example: hack/deploy-karmada-agent.sh ~/.kube/karmada.config karmada-apiserver ~/.kube/karmada.config member1"
+  echo "You could deploy karmada agent in any kubernetes clusters, e.g. karmada-host."
+  echo "Usage: hack/deploy-karmada-agent.sh <HOST_CLUSTER_KUBECONFIG> <HOST_CLUSTER_NAME> <KARMADA_APISERVER_KUBECONFIG> <KARMADA_APISERVER_CONTEXT_NAME> <MEMBER_CLUSTER_KUBECONFIG> <MEMBER_CLUSTER_NAME>"
```

Comment:
I suggest to add comments for params. eg, HOST_CLUSTER_KUBECONFIG is the kubeconfig path of the cluster wanted to deploy karmada agent.

Reply:
Too many parameters, isn't it?
Why do we require both `HOST_CLUSTER_KUBECONFIG` and `MEMBER_CLUSTER_KUBECONFIG`?
If people want to deploy `agent` on the member cluster, why they should provide the `HOST_CLUSTER_KUBECONFIG`?

Reply:
You are right, it is a little complicated. But if <HOST_CLUSTER_KUBECONFIG> is not specified, we don't know where to deploy the karmda-agent. So it only can be deployed into MEMBER_CLUSTER. But actually we don't have to do that. I don't find a better method.